src/lbaf/Execution/lbsInformAndTransferAlgorithm.py

```diff
@@ -315,8 +308,10 @@ def execute(self, phase: Phase, distributions: dict, statistics: dict, a_min_max
             n_ignored, n_transfers, n_rejects = self.transfer_stage()
             n_proposed = n_transfers + n_rejects
             if n_proposed:
-                self.__logger.info(f"{n_proposed} proposed transfers, {n_transfers} occurred, {n_rejects} rejected "
-                                   f"({100. * n_rejects / n_proposed:.4}%)")
+                self.__logger.info(
```

Comment:
@ppebay please make it as it was:

```python3
self.__logger.info(f"{n_proposed} proposed transfers, {n_transfers} occurred, {n_rejects} rejected "
                   f"({100. * n_rejects / n_proposed:.4}%)")
```

Reply:
Should be done now but with **explicit** concatenation rather than **implicit**.